### ISSUE TITLE
Decay/env mod not working?

### ISSUE THREAD

**Issue by simonvanderveldt** [DISTRHO/Nekobi#12]
Hi Falk,

Could it be the env mod/decay isn't working? I don't really hear any differences when setting higher decay values.

**Comment by falkTX** [comment]
@gordonjcp might know as he is the original author. but it should be working, plugin has been used by a lot of people, hard to think you are the first one to notice this.

**Comment by simonvanderveldt** [comment]
Yeah, agreed, I'd also be surprised if it's the case. I guess I can record some audio with env mod and decay at 0% and 100% and compare the waveforms. But I'd expect the difference to be audible.

**Comment by simonvanderveldt** [comment]
Did some quick tests with both the LV2 plugin as well as the standalone. Env mod does change the sound but decay doesn't seem to have any influence at all. So it seems like the envelope/decay is fixed.

**Comment by gordonjcp** [comment]
On Sat, Jan 02, 2021 at 04:08:10AM -0800, Simon van der Veldt wrote:
> Did some quick tests with both the LV2 plugin as well as the standalone. Env mod does change the sound but decay doesn't seem to have any influence at all. So it seems like the envelope/decay is fixed.
> 

If it works like the original plugin, then the decay knob ought to affect the VCF decay but the envelope decay is fixed at about two seconds.

The general idea was that it would be an approximation of a TB303, based on what I could remember of using a 303 many years before :-)

I now know a hell of a lot more about VA synthesis and would do just about all of it differently.

-- 
Gordonjcp
